pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, appimageTools }:
+
+let
+  pname = "Ripcord";
+  version = "0.4.18";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://cancel.fm/dl/${pname}-${version}-x86_64.AppImage";
+    sha256 = "19nv5pq8643l57hr5bdgn84rrw3ak2p9s16gr202nvi62flnl44p";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+in appimageTools.wrapType2 rec {
+  inherit name src;
+
+  extraInstallCommands = ''
```

Comment:
I think we should wrap the binary to set `RIPCORD_ALLOW_UPDATES=0` so it doesn't try to automatically update itself and fail

Reply:
afaik ripcord never attempts to self update. Ripcord only queries the update URL to show update notifications in Ripcord itself (which the `RIPCORD_ALLOW_UPDATES` disables).

Reply:
I personally would not like update notifications for software delivered via package manger. You end up in annoying situations where there's a new version released but it's not packaged yet so you get notified over and over.